README.md

```diff
@@ -411,3 +438,20 @@ template.hbs:
 
 {{load-more-button action='infinityLoad' infinityModel=model}}
 ```
+
+### Using ember-infinity with non-blocking model hooks
```

Comment:
I'd just call this "Non-Blocking Model Hooks" and ideally make it a h1 - as I think it's a big feature for people

Reply:
Gotcha.  In this case, it might make sense to put under the basic usage section?